README.md

```diff
@@ -1,79 +1,87 @@
-## Security
+# Preview: AWS Amplify's new code-first DX (Gen 2) for building backends
 
-See [CONTRIBUTING](CONTRIBUTING.md#security-issue-notifications) for more information.
+This next generation of Amplify’s backend building experience lets you author your frontend and backend definition completely with TypeScript, a file convention, and Git branch-based environments, is now available in Preview. To learn more, please visit [AWS Amplify (Gen 2)](https://docs.amplify.aws/gen2/).
 
-## License
+## Quickstart 
 
-This project is licensed under the Apache-2.0 License.
+To quickly get started with Amplify (Gen 2), please visit [AWS Amplify (Gen 2) Quickstart](https://docs.amplify.aws/gen2/start/quickstart/).
 
-## Set up your local development environment
+## Manual Installation
```

Comment:
Don't we have [manual installation](https://docs.amplify.aws/gen2/start/manual-installation/) in the docs? Keeping them in 2 places would cause issues to maintain sync. I'd say we should just point to the docs.

This readme should be perhaps more about what Amplify Gen2 is (doesn't change) rather than how to use it (changes often)

Reply:
Removed the reference to the manual installation.